src/router/routes.js

```diff
@@ -121,6 +121,15 @@ module.exports = [
 		path: '/donate/supportus',
 		component: () => import('@/pages/Donate/DonateSupportUs'),
 		meta: {
+			contentfulPage: () => 'donate/supportus',
+			unbounce: true,
+		},
+	},
+	{
+		path: '/donate/supportkiva',
+		component: () => import('@/pages/Donate/DonateSupportUs'),
+		meta: {
+			contentfulPage: () => 'donate/supportkiva',
```

Comment:
Maybe a bit verbose, but it's clear what the contentful key is. I could've also gone with just setting the key to the path....